packages/webapp/src/components/Task/PureTaskTypeSelection/PureTaskTypeSelection.jsx

```diff
@@ -77,6 +79,9 @@ export const PureTaskTypeSelection = ({
   register(TASK_TYPE_ID);
   const selected_task_type = watch(TASK_TYPE_ID);
 
+  const isMakingCropTask = location?.state?.management_plan_id ?? false;
+  const animalTasks = ['MOVEMENT_TASK'];
```

Comment:
I have a feeling we might reuse this, so it would be nice to store it somewhere like `packages/webapp/src/containers/Task/constants.js`.

Reply:
Good call, moved!